### PR TITLE
Prac 08 code review request

### PR DESCRIPTION
**Pull request by Elsa-Stapelbroek**
Hi @jessepurcell ! Would you be able to review my code for last week's practical?

### CODE REVIEW

File: prac_08/convert_miles_km.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+"""CP1404 practical 8 - Do-from-scratch 1: Miles to Kilometres Converter"""
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.properties import StringProperty
+
+MILES_TO_KM_FACTOR = 1.609344
```
Comment (jessepurcell) [inline prac_08/convert_miles_km.py:6]: Good use of constant

File: prac_08/box_layout_demo.py
Comment (jessepurcell) [inline prac_08/box_layout_demo.py:1]: Missing method docstrings

File: prac_08/convert_miles_km.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+class MilesConversionApp(App):
+    """Kivy App for converting miles to kilometres."""
+    result = StringProperty()
```
Comment (jessepurcell) [inline prac_08/convert_miles_km.py:11]: Good use of a StringProperty()

**Review by jessepurcell — COMMENTED** [review]
Good docstrings and error handling. Only issue I can see is missing method docstrings in prac_08/box_layout_demo.py
Looks like you understand the concepts of this weeks pracs 👍

File: prac_08/convert_miles_km.py (hunk 2 above)
Comment (Elsa-Stapelbroek) [reply prac_08/convert_miles_km.py:11]: Thank you! That one definitely took some time to figure out.

File: prac_08/box_layout_demo.py
Comment (Elsa-Stapelbroek) [reply prac_08/box_layout_demo.py:1]: Good point! They should be there now :)

**Pull request merged**